EKSD_LATEST_RELEASES

```diff
@@ -23,4 +23,8 @@ releases:
 - branch: 1-25
   number: 5
   kubeVersion: v1.25.6
+- branch: 1-26
+  number: 1
+  kubeVersion: v1.26.0
+  dev: true
 latest: 1-22
```

Comment:
We need to have this conversation again about bumping this up...

Reply:
i figured we could add the dev version now & we should have the 1.26 prod version before the next release so it wont be pulling from dev in the bundle